indico/modules/rb/controllers/backend/bookings.py

```diff
@@ -225,9 +225,15 @@ def reject(self, reason):
         self.booking.reject(session.user, reason)
 
     @use_kwargs({
-        'reason': fields.String(required=False)
+        'reason': fields.String(required=False),
+        'force': fields.Bool(required=False)
     })
-    def accept(self, reason=None):
+    def accept(self, reason=None, force=False):
+        if not force:
+            collisions = get_prebooking_collisions(self.booking)
+            collision_data = reservation_occurrences_schema.dump(collisions)
+            if collision_data:
```

Comment:
I'd check `if collisions` and only dump inside the if when you need the jsonifiable data